src/client/components/PageLayout/Toolbar/Status/StatusConfirm/StatusConfirm.tsx

```diff
@@ -42,54 +34,36 @@ const StatusConfirm: React.FC<Props> = (props) => {
   const [textareaValue, setTextareaValue] = useState<string>('')
   const { assessmentName, cycleName } = useParams<{ assessmentName: AssessmentName; cycleName: string }>()
 
-  const recipients = useTablePaginatedData({ path: `${ApiEndPoint.User.many()}#recipients` })
-  const hasRecipients = !Objects.isEmpty(recipients)
-
   return (
     <Modal isOpen>
       <ModalHeader>
-        <div className="modal-header-center">{i18n.t(`assessment.status.${status.status}.${status.direction}`)}</div>
+        <div className="modal-header-center">{t(`assessment.status.${status.status}.${status.direction}`)}</div>
         <ModalClose onClose={onClose} />
       </ModalHeader>
 
-      <ModalBody>
+      <ModalBody className="assessment-status-confirm__body">
         <div style={{ height: '160px' }}>
           <textarea
-            className="nav-assessment-status-confirm__message"
+            className="assessment-status-confirm__message"
             onChange={({ target: { value } }): void => setTextareaValue(value)}
-            placeholder={i18n.t('navigation.changeStatusTextPlaceholder')}
+            placeholder={t('navigation.changeStatusTextPlaceholder')}
             value={textareaValue}
           />
         </div>
 
-        {status.status !== CountryStatus.approval && Users.isAdministrator(user) && hasRecipients && (
-          <div
-            className="nav-assessment-status-confirm__notify-users"
-            onClick={(): void => setNotifyUsers(!notifyUsers)}
-            onKeyDown={(): void => setNotifyUsers(!notifyUsers)}
-            role="button"
-            tabIndex={0}
-          >
-            <div className={classNames('fra-checkbox', { checked: !notifyUsers })} />
-            {i18n.t('navigation.doNotNotifyUsers')}
-          </div>
-        )}
-        {notifyUsers && <UserList status={status} />}
-        <div
-          className="nav-assessment-status-confirm__notify-self"
+        <NotifyUsers notifyUsers={notifyUsers} setNotifyUsers={setNotifyUsers} status={status} />
```

Comment:
when admin clicked on Don't notify Users. the user list and checkbox disappeared completely without giving the possibility to rollback the selection.